### PR TITLE
Update installation guide to include submodules

### LINKED ISSUE #5408: update the clone step in the installation guide to include submodules
## Details

When running the pre-commit command `pre-commit run`, it gives the error that `.pre-commit-config.yaml` does not exist though the command `ls -a` clearly mentions that `.pre-commit-config.yaml` exists.

## Actual Result

screenshot from 2019-03-07 01-23-19

### PR DESCRIPTION
**Pull request by agarwalrounak**
Fixes #5408 
The clone step in the installation guide is updated to include submodules.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -77,9 +77,9 @@ you need these libraries.
 Get and run Read the Docs
 -------------------------
 
-Clone the repository somewhere on your disk and enter to the repository::
+Clone the repository somewhere on your disk, initialize and update the submodule, and enter to the repository::
```
Comment (stsewd) [inline docs/install.rst]: We can remove this sentence now I think, this is done implicitly.

**Comment by agarwalrounak** [comment]
Thanks for the review, learned a new thing today.

File: docs/install.rst (hunk 1 above)
Comment (stsewd) [reply docs/install.rst]: (the sentence about submodules only)

Comment (agarwalrounak) [reply docs/install.rst]: Done!

**Comment by agarwalrounak** [comment]
Can you please take a look at PR #5411

**Comment by stsewd** [comment]
Tests need to pass first before any review

**Pull request merged**